content/docs/guides/ha_scale/performace.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Open Service Mesh Performance"
+description: "Performance benchmark on control plane and data plane sidecar"
+type: docs
+---
+
+# Performance
+
+OSM aims to create a high performant service mesh while providing all the features. The development team has done a thorough benchmark on its performance.
+
+## Benchmark Setup
```

Comment:
The setup must describe the cluster configuration - number of workers, number of cores, CPU, RAM of nodes etc.